instructions/3-create-postgres-on-azure.md

```diff
@@ -0,0 +1,7 @@
+# Create and Configure PostgreSQL on Azure
+
+- Overview of Postgres pricing tiers
+- Exercise: Create Postgres DB with Azure CLI
+- Exercise: Update the JBoss app for Postgres

```

Comment:
Is there a way to use `az webapp deploy` to create the needed CLI, postgres module jar file, and startup.sh script(s) easily? rather than manually FTPing them in place?

Reply:
Yeah we can use `az webapp deploy` or the Maven plugin to deploy all those files, under the hood they use the same API